packages/webview_flutter/webview_flutter_web/lib/webview_flutter_web.dart

```diff
@@ -183,7 +184,11 @@ class WebWebViewPlatformController implements WebViewPlatformController {
     String? baseUrl,
   }) async {
     // ignore: unsafe_html
-    _element.src = 'data:text/html,${Uri.encodeFull(html)}';
+    _element.src = Uri.dataFromString(
+      html,
+      mimeType: 'text/html',
+      encoding: Encoding.getByName('utf-8'),
```

Comment:
Can this use the `utf8` top-level constant [(docs)](https://api.dart.dev/stable/2.17.3/dart-convert/utf8-constant.html) instead of `Encoding.getByName`?